package.json

```diff
@@ -19,23 +19,42 @@
       "feesfines": "16.0 17.0 18.0 19.0",
       "data-export-spring": "1.0"
     },
-    "stripesDeps": [
```

Comment:
why is it removed?

Reply:
if you completely don't use stripes-acq-components, please remove dependency as well